assets/images/README.md

```diff
@@ -0,0 +1,5 @@
+# Images🖼️
+The `/assets/images` directory holds all the site's images.
+
+
+See the [contributing docs](/CONTRIBUTING.md) for general information about working with the docs.
```

Comment:
```suggestion
See [imaging and versioning](https://github.com/github/docs/blob/main/contributing/images-and-versioning.md) from the contributing docs for more information.
```
Awesome job linking users to the contributing.md ⚡
It would be great to take it just that extra step further and link them to the actual imaging md, so I've adjusted the link.